docs/ensnode.io/src/styles/starlight.css

```
@@ -696,12 +726,114 @@ starlight-tabs ul[role="tablist"] {
+  font-size: 0.75rem;
+}
+
+.sql-result-cell-pre {
+  margin: 0;
+  background: none;
+  padding: 0;
+  font-size: 0.75rem;
+  line-height: 1.3;
+  color: inherit;
+  white-space: pre-wrap;
+  word-break: break-word;
```

Stylelint (17.12.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)